pkg/util/docker/docker_util.go

```diff
@@ -161,10 +161,18 @@ func (d *DockerUtil) GetStorageStats() ([]*StorageStats, error) {
 	return parseStorageStatsFromInfo(info)
 }
 
+func isImageShaID(image string) bool {
+	return strings.HasPrefix(image, "sha256:")
+}
+
+func isImageRepoDigest(image string) bool {
+	return strings.Contains(image, "@sha256:")
+}
```

Comment:
do you think we could merge these helpers into the latter, since they seem to be used together?

Reply:
Yes, thank you, we can combine them into one.